styles/widgets/generic/dropDownButton.generic.less

```diff
@@ -2,8 +2,18 @@
 @import (once) "./popup.generic.less";
 @import (once) "./list.generic.less";
 
+.dx-button {
```

Comment:
This style will be applied for all buttons, will not?

Reply:
fixed it